src/payment/strategies/paypal-commerce/paypal-commerce-sdk.ts

```diff
@@ -54,10 +59,18 @@ export interface PaypalCommerceHostWindow extends Window {
     paypal?: PaypalCommerceSDK;
 }
 
+export interface PaypalCommerceInitializationData {
+    clientId: string;
+    intent?: 'capture' | 'authorize';
+    isPayPalCreditAvailable?: boolean;
+}
+
+export type disableFundingType = Array<'credit' | 'card'>;
```

Comment:
Can you capitalise this as you're defining a new type?

Reply:
Sure. Thank you!